index.html

```diff
@@ -3,16 +3,22 @@
   <head>
     <title>"ALL about me"</title>
     <meta name="Farida" content="HTML"/>
+    <link rel="stylesheet" href = "css/index.css">
+   
   </head>
   <body>
+
+    <section id="my name" style="text-align: center;">
 <h1>Farida Hyderi</h1>
+</section>
+
     <section id="About">
 <h2>About</h2>
-<p>My name is Farida I am very entusiastic to learn new programming langusges
+<p style= "font-family: Times New Roman; ">My name is Farida I am very entusiastic to learn new programming languages
 also very much interested in testing </p>
     </section>
 
-    <section id = "Experience">
+    <section id = "Experience" >
 <h2>Experience</h2>
       <ul>
 <li>Cognizant Agile Methodology from Forage </li>
```

Comment:
The assignment instructions mention putting your experience "in stylized blocks". You can do this with `<divs>`. Take a look [here](https://www.theodinproject.com/lessons/foundations-block-and-inline#divs-and-spans) for an example.